recipe/meta.yaml

```
@@ -30,11 +33,19 @@ requirements:
     - more-itertools
     - zc.lockfile
     - jaraco.collections
-    - pywin32 >=227   # [win and py<310]
-    - importlib-metadata # [py<38]
-    - simplejson
+    - pywin32 >=227  # [win and py<310]
```

```suggestion
    - pywin32 >=227  # [win and py<310]
    - pyopenssl
```
It's optional, but it is good to keep it as default dependency.
https://github.com/cherrypy/cherrypy/blob/v18.10.0/setup.py#L86